src/test/java/com/yodle/vantage/functional/VersionCreateAndUpdateTest.java

```diff
@@ -162,6 +162,23 @@ public void updateVersionWithDepsToHaveMoreDeps() {
     }
 
     @Test
+    public void updateVersionWithRequestedDepsToHaveMoreDeps() {
+        Dependency dependency = createDependency("dependency component", "1.0.+");
+        createVersion("component", "version", Sets.newHashSet(dependency));
+
+        //add a profile to an existing dep
+        dependency.getProfiles().add("new profile");
+        dependency.getVersion().setVersion("1.+");
```

Comment:
dynamic versions like this get stored as 'unknown', but we store the value requested as a property on the REQUESTS relationship.